Description: Avoid confusion of Linitian with donation HTML code in comments
 I simply remove the lines from the comment. The override rules keep failing.
Author: Markus Frosch <lazyfrosch@debian.org>
Last-Update: 2014-09-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/app/modules/AppKit/lib/js/utils/HttpStateProvider.js
+++ b/app/modules/AppKit/lib/js/utils/HttpStateProvider.js
@@ -75,15 +75,6 @@
  *
  * @forum     24970
  * @demo      http://cellactions.extjs.eu
- *
- * @donate
- * <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank">
- * <input type="hidden" name="cmd" value="_s-xclick">
- * <input type="hidden" name="hosted_button_id" value="3430419">
- * <input type="image" src="https://www.paypal.com/en_US/i/btn/x-click-butcc-donate.gif" 
- * border="0" name="submit" alt="PayPal - The safer, easier way to pay online.">
- * <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
- * </form>
  */
 
 Ext.ns('Ext.ux.state');
@@ -466,4 +457,4 @@
 
 }); // eo extend
 
-// eof
\ No newline at end of file
+// eof
